src/components/DatePicker/index.js

```diff
@@ -31,9 +31,6 @@ class DatePicker extends React.Component {
         // Adds nice native datepicker on web/desktop. Not possible to set this through props
         this.inputRef.setAttribute('type', 'date');
         this.inputRef.classList.add('expensify-datepicker');
-        if (this.props.maximumDate) {
-            this.inputRef.setAttribute('max', moment(this.props.maximumDate).format(CONST.DATE.MOMENT_FORMAT_STRING));
-        }
```

Comment:
These lines lead me to wonder what happens if we don't supply a maximumDate prop. 
Is it necessary to remove the prop all together?
Maybe we could default the behavior to have no maximum date unless one is supplied.

Reply:
I'm not sure that I understand the concern. All this line does is also set the `max` attribute for the [type = date](https://github.com/Expensify/App/blob/4f2ef4cf737c8d406c82a656615ba3fe6c16bf19/src/components/DatePicker/index.js#L32) input we set here. The default behavior was already [undefined](https://github.com/Expensify/App/pull/13702/files#diff-33e81c8b58e44ffc9cd1a0528ef9e2c8654f54d2a9a98962ecc2bc85f594b17eL29), but if we never set `maximumDate` there's no need to keep this around.

Reply:
Hmm, yeah I suppose that's true. If we're not using it, then we probably should just get rid of it. 
Thx 👍